Basic/Invaders/Assets/Prefabs/bullet.prefab

```diff
@@ -10,10 +10,10 @@ GameObject:
   m_Component:
   - component: {fileID: 400000}
   - component: {fileID: 21200000}
+  - component: {fileID: 2859882261451710602}
```

Comment:
@bastienunity  it seems the order of scripts was changed here as well, is this intended? If not, can you revert it to avoid potential issues?

Reply:
So, funny story... When selecting the prefab, the Inspector refreshes after one frame and moves the NetworkObject component in third place on every single Prefab I select in the project. So any further changes made to the prefab will save the NetworkObject component position changed...
I can manually revert that change outside of Unity, but because it is forced and automatic, I think it's better to just commit it?

Reply:
If Netcode is forcing reserialization, and we're going to keep running into this change in subsequent PRs, this can go through.
Let's expose our code to potential race conditions and address them, if they arise!

Reply:
@RikuTheFuffs

Reply:
yeah, if it's automatic there's little we can do. I just wanted to be sure that it was not a manual change.